### ISSUE TITLE
Idea: args shortening

### ISSUE THREAD

**Issue by pvalena** [psss/tmt#161]
Can we do shortening like `--playbook` = `--playbooks`, `--script` = `--scripts` etc. then?

_ _ _ _

    tmt plan --hel
    Usage: tmt plan [OPTIONS] COMMAND [ARGS]...
    Try "tmt plan --help" for help.

    Error: no such option: --hel  Did you mean --help?

For command shortening I needed to do [this](https://github.com/psss/tmt/blob/master/tmt/cli.py#L29).

_Originally posted by @psss in https://github.com/psss/tmt/timeline_

**Comment by psss** [comment]
Yeah, definitely +1. We already have a [story](https://tmt.readthedocs.io/en/latest/stories/cli.html#options) for this. Just to implemet it ;-)
